compiler/arser/CMakeLists.txt

```diff
@@ -12,4 +13,4 @@ endif(NOT ENABLE_TEST)
 nnas_find_package(GTest REQUIRED)
 set(TESTS "${CMAKE_CURRENT_SOURCE_DIR}/tests/arser.test.cpp")
 GTest_AddTest(arser_test ${TESTS})
-target_include_directories(arser_test PRIVATE include)
+target_link_libraries(arser_test arser)
```

Comment:
I'm not sure but does it have same meaning?

Reply:
Yup, `arser` has `target_include_directories(arser INTERFACE include/)` and linking `arser` will include to `include/` folder